server/aurora_mcp/registry.py

```
@@ -0,0 +1,789 @@
+def _check_skill_connected(skill_id: str, user_id: str) -> bool:
+    """Return True iff `skill_id` is connected for `user_id`.
+
+    Lazy-imports SkillRegistry to keep this module import-light.
+    Failures (missing skill, registry errors) are treated as disconnected.
+    """
+    try:
+        from chat.backend.agent.skills.registry import SkillRegistry
+
+        is_connected, _ = SkillRegistry.get_instance().check_connection(skill_id, user_id)
+        return bool(is_connected)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)